cinema/views.py

```diff
@@ -11,7 +12,7 @@
     MovieSessionListSerializer,
     MovieDetailSerializer,
     MovieSessionDetailSerializer,
-    MovieListSerializer,
+    MovieListSerializer, OrderSerializer, OrderListSerializer,
 )
```

Comment:
Code Style: Group imports using `()` if needed. The import statement for the serializers is not grouped correctly. The trailing comma should be removed from the last import inside the parentheses.